config/locales/en.yml

```diff
@@ -64,3 +64,12 @@ en:
     move_stockit: "You can only move it using Stockit."
   orders_package:
     already_dispatched: "This has been already dispatched."
+  organisations_user_builder:
+    organisation:
+      blank: "Organisation can't be blank"
+      not_found: "Organisation not found"
+    user:
+      mobile:
+        blank: "Mobile can't be blank"
+    existing_user:
+      present: "User's already exist in organisation"
```

Comment:
Could we change the English slightly here - "User already exists in this organisation". Thanks!